package.json

```diff
@@ -34,11 +34,11 @@
     "create-types-lf-ui-components": "npm run tsc-cdn && npm run api-extractor && gulp processTypesFile && npm run move-enums-to-types && cd ./types-lf-ui-components-publish && npm pack"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.12",
+    "@angular-devkit/build-angular": "^15.2.9",
     "@angular-eslint/builder": "^14.4.0",
```

Comment:
I think the @angular-eslint packages should actually be updated as well

Reply:
Just updated